cores/esp8266/core_esp8266_i2s.c

```diff
@@ -218,7 +218,7 @@ void ICACHE_FLASH_ATTR i2s_set_rate(uint32_t rate){ //Rate in HZ
 
   //!trans master, !bits mod, rece slave mod, rece msb shift, right first, msb right
   I2SC &= ~(I2STSM | (I2SBMM << I2SBM) | (I2SBDM << I2SBD) | (I2SCDM << I2SCD));
-  I2SC |= I2SRF | I2SMR | I2SRSM | I2SRMS | ((i2s_bck_div-1) << I2SBD) | ((i2s_clock_div-1) << I2SCD);
+  I2SC |= I2SRF | I2SMR | I2SRSM | I2SRMS | ((i2s_bck_div) << I2SBD) | ((i2s_clock_div) << I2SCD);
```

Comment:
Looking at the math in this function vs. what's in the MP3_decoder vs. what's there now, I can't see how increasing I2SBASEFREQ by 13.33x and then incrementing the divider (aka div-by-2) would work.  Assuming it works at all, wouldn't this just increase the output frequency by 13.33/2 or 13.33/4?

The MP3 decoder reference has a series of configs on the clock source for I2S and then a large for-loop to determine the closest divider with their (different, I believe) clock inputs.
https://github.com/espressif/ESP8266_MP3_DECODER/blob/7552a62d425598d64ebc255d32fa4f20220e92f6/mp3/driver/i2s_freertos.c#L250

Have you run this with a frequency counter on LRCLK for different CPU freqs and sample rates?  The existing logic is close enough for most sample outputs I've seen for multi-hour AAC and MP3 streams, so seeing such a large change in clocking requested here really puzzles me.